src/app/myworkflows/my-workflow/my-workflow.component.html

```diff
@@ -21,20 +21,12 @@
 
   <div fxLayout="column" fxFlex>
     <app-header fxFlex>
-      <div fxLayout="row" fxLayoutAlign="start center">
+      <div fxLayout fxLayoutAlign="start center">
         <button mat-icon-button (click)="toggleSidebar()">
-          <mat-icon *ngIf="showSidebar">chevron_left</mat-icon>
-          <mat-icon *ngIf="!showSidebar">chevron_right</mat-icon>
+          <mat-icon>chevron_{{ showSidebar ? 'left' : 'right' }}</mat-icon>
         </button>
-        <div *ngIf="(myEntryPageTitle$ | async | titlecase) === 'My Workflows'" fxLayoutGap="0.5rem">
-          <img src="../assets/svg/sub-nav/workflow.svg" alt="workflow icon" />
-          <span>{{ myEntryPageTitle$ | async | titlecase }}</span>
-        </div>
-        <div *ngIf="(myEntryPageTitle$ | async | titlecase) === 'My Services'" fxLayoutGap="0.5rem">
-          <img src="../assets/svg/sub-nav/services.svg" alt="service icon" />
-          <span>{{ myEntryPageTitle$ | async | titlecase }}</span>
-          <span class="beta-label"> preview </span>
-        </div>
+        <span>{{ myEntryPageTitle$ | async | titlecase }}</span>
+        <span *ngIf="(myEntryPageTitle$ | async | titlecase) === 'My Dockstore: Services'" class="beta-label ml-1"> preview </span>
```

Comment:
This `preview` label is different from the one that the services entry box uses on the new dashboard. Perhaps we should update this one to use the new preview label?